tests/eval_files/trio100.py

```diff
@@ -3,20 +3,23 @@
 import trio
 
 with trio.move_on_after(10):  # error: 5, "trio", "move_on_after"
-    pass
+    ...
```

Comment:
replaced `pass` with ellipsis partly in attempts to avoid shed headaches, but means you can see a difference with ellipsis's transferred over, vs injected `pass`s, in the autofix file.